WindowsClient/_Data/_Actions/FortranChecker.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WindowsClient._Data._Helpers;
+
+namespace WindowsClient._Data._Actions
+{
+    public static class Fortran
+    {
+        public static readonly string PATH = @"C:\";
+
+        public static bool Exists(string path)
+        {
+            return File.Exists(path);
+        }
+
+        public static void Execute(string path, string name = "gfortran-windows.exe")
```

Comment:
its work in fortran77 or all fortran types? if i have other compiler... what is the exception?